src/app/app.js

```diff
@@ -146,6 +147,19 @@ angular
       })
       .when('/censusdata/create', {
         templateUrl: 'app/views/censusdata/create.html',
+        //TODO: Will think about should conroller be seperated out for each of these views.
+        controller: 'CensusDataCreateController',
```

Comment:
Well... let me think about that more. 1 problem with separate controllers is that each controller is another file to add to index.html.